README.md

```diff
@@ -619,6 +620,27 @@ setCustomerHash(
 );
 ```
 
+### getDiskSpaceInfo
+
+Return info about how much free disk space there is in the device
```

Comment:
this phrase sounds weird to me  xDD
'how much free disk space there is in the device' is it correct? just talking about english xDD
how much free disk space the device has?

Reply:
lol, yes, you're right, I'll change this